test/llvm/abi/aarch64/export-vs-c-llvm18.good

```diff
@@ -0,0 +1,206 @@
+OUTPUT testing struct_triple_..._c_____: arg.a 0 arg.b 1 arg.c 2
+OUTPUT testing struct_triple_..._chapel
+OUTPUT testing struct_triple_..._chapel: arg.a 0 arg.b 1 arg.c 2
+OUTPUT testing struct_twelve_..._c_____
+OUTPUT testing struct_twelve_..._c_____: arg.a 0 arg.b 1 arg.c 2 arg.d 3 arg.e 4 arg.f 5 arg.g 6 arg.h 7 arg.i 8 arg.j 9 arg.k 10 arg.l 11
+OUTPUT testing struct_twelve_..._chapel
+OUTPUT testing struct_twelve_..._chapel: arg.a 0 arg.b 1 arg.c 2 arg.d 3 arg.e 4 arg.f 5 arg.g 6 arg.h 7 arg.i 8 arg.j 9 arg.k 10 arg.l 11
+OUTPUT testing uint16_..._c_____
+OUTPUT testing uint16_..._c_____: arg 65535
+OUTPUT testing uint16_..._chapel
+OUTPUT testing uint16_..._chapel: arg 65535
+OUTPUT testing uint32_..._c_____
+OUTPUT testing uint32_..._c_____: arg 4294967295
+OUTPUT testing uint32_..._chapel
+OUTPUT testing uint32_..._chapel: arg 4294967295
+OUTPUT testing uint64_..._c_____
+OUTPUT testing uint64_..._c_____: arg 18446744073709551615
+OUTPUT testing uint64_..._chapel
+OUTPUT testing uint64_..._chapel: arg 18446744073709551615
+OUTPUT testing uint8_..._c_____
+OUTPUT testing uint8_..._c_____: arg 255
+OUTPUT testing uint8_..._chapel
+OUTPUT testing uint8_..._chapel: arg 255
+define %struct.c_one_double @struct_one_double_return_c_____()
+define %struct.c_one_double @struct_one_double_return_chapel()
+define %struct.c_one_float @struct_one_float_return_c_____()
+define %struct.c_one_float @struct_one_float_return_chapel()
+define %struct.c_pair_double @struct_pair_double_return_c_____()
+define %struct.c_pair_double @struct_pair_double_return_chapel()
+define %struct.c_pair_float @struct_pair_float_return_c_____()
+define %struct.c_pair_float @struct_pair_float_return_chapel()
+define [2 x i64] @struct_paddingtest_return_c_____()
+define [2 x i64] @struct_paddingtest_return_chapel()
+define [2 x i64] @struct_pair_long_return_c_____()
+define [2 x i64] @struct_pair_long_return_chapel()
+define double @double_return_c_____()
+define double @double_return_chapel()
+define float @float_return_c_____()
+define float @float_return_chapel()
+define i32 @int32_return_c_____()
+define i32 @int32_return_chapel()
+define i32 @struct_one_int_return_c_____()
+define i32 @struct_one_int_return_chapel()
+define i32 @uint32_return_c_____()
+define i32 @uint32_return_chapel()
+define i64 @int64_return_c_____()
+define i64 @int64_return_chapel()
+define i64 @struct_one_long_return_c_____()
+define i64 @struct_one_long_return_chapel()
+define i64 @struct_pair_int_return_c_____()
+define i64 @struct_pair_int_return_chapel()
+define i64 @uint64_return_c_____()
+define i64 @uint64_return_chapel()
+define signext i16 @int16_return_c_____()
+define signext i16 @int16_return_chapel()
+define signext i8 @int8_return_c_____()
+define signext i8 @int8_return_chapel()
+define void @abistructfunc_c_____(i32 noundef, i32 noundef, [2 x i64], i32 noundef, i32 noundef, double noundef, double noundef, double noundef, i32 noundef, i32 noundef, i32 noundef0)
```

Comment:
arguably, we should be adding `noundef`. If you don't know a reason why we're not, I think this would be worth an issue if we haven't created one about this already. note that this issue also applies to x86-64 but was present on `main` already with LLVM 17.